recipes/libtorch/all/conanfile.py

```diff
@@ -0,0 +1,667 @@
+        "with_kineto": "Use Kineto profiling library",
+        "with_magma": "Use MAGMA linear algebra library",
+        "with_metal": "Use Metal for iOS build",
+        "with_mimalloc": "Use mimalloc",
+        "with_mkldnn": "Use MKLDNN. Only available on x86, x86_64, and AArch64.",
+        "with_mps": "Use MPS for macOS build",
+        "with_nccl": "Use NCCL and RCCL",
+        "with_nnapi": "Use NNAPI for Android build",
+        "with_nnpack": "Use NNPACK CPU acceleration library",
+        "with_numa": "Use NUMA. Only available on Linux.",
+        "with_nvrtc": "Use NVRTC",
+        "with_opencl": "Use OpenCL",
+        "with_openmp": "Use OpenMP for parallel code",
+        "with_qnnpack": "Use ATen/QNNPACK (quantized 8-bit operators)",
+        "with_rocm": "Use ROCm (HIP)",
+        "with_snpe": "Use Qualcomm's SNPE library",
+        "with_vulkan": "Use Vulkan GPU backend",
+        "with_xnnpack": "Use XNNPACK",
+        "with_xpu": "Use XPU (SYCL) backend for Intel GPUs",
+    }
+    no_copy_source = True
+    provides = ["miniz", "pocketfft", "kineto", "nnpack", "qnnpack"]
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "5",
+            "apple-clang": "9",
+            "msvc": "191",
+            "Visual Studio": "15",
+        }
+
+    @property
+    def _is_mobile_os(self):
+        return self.settings.os in ["Android", "iOS"]
+
+    def export_sources(self):
+        export_conandata_patches(self)
+        copy(self, "conan_deps.cmake", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
+        copy(self, "conan-official-libtorch-vars.cmake", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            del self.options.with_qnnpack
+        if not is_apple_os(self):
+            self.options.rm_safe("with_metal")
+            self.options.rm_safe("with_magma")
+            self.options.rm_safe("with_mps")
+        if self.settings.os != "Linux":
+            del self.options.with_numa
+        if self.settings.os != "Android":
+            self.options.rm_safe("with_nnapi")
+            self.options.rm_safe("with_snpe")
+            self.options.with_vulkan = False
+        if self._is_mobile_os:
+            self.options.blas = "eigen"
+            self.options.build_lazy_ts_backend = False
+            del self.options.with_fbgemm
+            # del self.options.distributed
+        if self.settings.arch not in ["x86", "x86_64", "armv8"]:
+            self.options.rm_safe("with_mkldnn")
+        if not is_apple_os(self) or self.settings.os not in ["Linux", "Android"]:
+            del self.options.with_nnpack
+        self.options.with_itt = self.settings.arch in ["x86", "x86_64"]
```

Comment:
This line overrides the manual taken settings in `with_itt`. So even if set to `False` this line sets it to `True` on x86, x86_64 which may not be the expected behaviour.